src/Microsoft.Tye.Hosting/Model/JSON/ServiceDescriptionJson.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Tye.Hosting.Model
+{
+    public class ServiceDescriptionJson
+    {
+        public string? Name { get; set; }
```

Comment:
I realize I'm currently not serializing/deserializing RunInfo. I'll take a stab at it now, but it's a bit tricky with all the subclasses.